tests/test_hookman_utils.py

```diff
@@ -8,3 +8,16 @@ def test_find_config_files(datadir):
 
     config_files = find_config_files([datadir / 'non_existing_folder'])
     assert len(config_files) == 0
+
+
+def test_load_shared_lib(simple_plugin_dll):
+    import os
+    import sys
+    from hookman.hookman_utils import load_shared_lib
+
+    if sys.platform.startswith('win'):
+        assert not str(simple_plugin_dll.parent) in os.environ["PATH"]
+        with load_shared_lib(str(simple_plugin_dll)):
+            pass
+
+        assert str(simple_plugin_dll.parent) in os.environ["PATH"]
```

Comment:
```suggestion
        paths = os.environ["PATH"].split(os.pathsep)
        assert Path(paths[0]) == simple_plugin_dll.parent
```

We want to also ensure we are putting the dll directory in front of `PATH`.

Reply:
Nice suggestion. Thanks!